media/static-bundles.json

```diff
@@ -1253,14 +1236,12 @@
       "files": [
         "js/base/send-to-device.js",
         "js/libs/spin.min.js",
-        "js/base/uitour-lib.js",
```

Comment:
You may need to rebase this branch, but I still see an stray uitour-lib.js on line 294:

```
    {
      "files": [
        "js/base/uitour-lib.js",
        "js/firefox/new/desktop/download.js"
      ],
      "name": "firefox_desktop_download"
    },
```